Cargo.toml

```diff
@@ -18,5 +18,5 @@ futures-util = "0.3.30"
 serde = { version = "1.0", features = ["derive"] }
 thiserror = "1.0"
 time = { version = "0.3", features = ["parsing"] }
-zbus = { version = "3.15.0" }
-zvariant = { version = "3.15.0"}
+zbus = { version = "4.1.2" }
```

Comment:
This should be 4.2.1. There are some critical bugs in zbus versions before 4.2.1

Reply:
Ok, gotcha